packages/components/button/src/Button.stories.tsx

```diff
@@ -73,8 +75,34 @@ export const Intent: StoryFn = _args => (
   </div>
 )
 
+export const Icons: StoryFn = _args => (
+  <div className="gap-md flex flex-wrap">
+    <Button>
+      Button
+      <Icon>
+        <FavoriteOutline />
+      </Icon>
+    </Button>
+    <Button>
+      <Icon>
```

Comment:
Not sure if we should provide a `Button.Icon` component for this. It will be just a wrapper of the `Icon`

Reply:
I don't think it's necessary but I'm not opposed to it either.